src/lib/types/Session.ts

```diff
@@ -16,4 +16,5 @@ export interface Session extends Timestamps {
 		name: string;
 		id: string;
 	}>;
+	clientIp?: string;
```

Comment:
Let's store `pos?: { countryCodeOverwrite?: ...}` instead

Let's not store a client ip in the session or mess with ips.

The `ipFromCountry` is a nice bit of code, and was fun to write probably, but boring code/simplest solution is better for long term :pray:

Reply:
So what do you mean ? There's no need to store the IP ?
If so how are we gonna the clientIP ?


(I took Inspiration on `countryFromIP`)